diff -urEb aws-c-io-0.14.9.orig/include/aws/io/private/pki_utils.h aws-c-io-0.14.9/include/aws/io/private/pki_utils.h
--- aws-c-io-0.14.9.orig/include/aws/io/private/pki_utils.h	2024-06-04 18:27:58.000000000 +0200
+++ aws-c-io-0.14.9/include/aws/io/private/pki_utils.h	2024-06-23 10:39:15.047978000 +0200
@@ -9,7 +9,7 @@
 #ifdef _WIN32
 /* It's ok to include external headers because this is a PRIVATE header file
  * (it is usually a crime to include windows.h from header file) */
-#    include <Windows.h>
+#    include <windows.h>
 #endif /* _WIN32 */
 
 #ifdef AWS_OS_APPLE
diff -urEb aws-c-io-0.14.9.orig/source/windows/iocp/iocp_event_loop.c aws-c-io-0.14.9/source/windows/iocp/iocp_event_loop.c
--- aws-c-io-0.14.9.orig/source/windows/iocp/iocp_event_loop.c	2024-06-04 18:27:58.000000000 +0200
+++ aws-c-io-0.14.9/source/windows/iocp/iocp_event_loop.c	2024-06-23 10:39:15.047978000 +0200
@@ -12,7 +12,7 @@
 
 #include <aws/io/logging.h>
 
-#include <Windows.h>
+#include <windows.h>
 
 /* The next set of struct definitions are taken directly from the
     windows documentation. We can't include the header files directly
diff -urEb aws-c-io-0.14.9.orig/source/windows/iocp/pipe.c aws-c-io-0.14.9/source/windows/iocp/pipe.c
--- aws-c-io-0.14.9.orig/source/windows/iocp/pipe.c	2024-06-04 18:27:58.000000000 +0200
+++ aws-c-io-0.14.9/source/windows/iocp/pipe.c	2024-06-23 10:39:15.047978000 +0200
@@ -11,7 +11,7 @@
 #include <stdbool.h>
 #include <stdio.h>
 
-#include <Windows.h>
+#include <windows.h>
 
 enum read_end_state {
     /* Pipe is open. */
diff -urEb aws-c-io-0.14.9.orig/source/windows/iocp/socket.c aws-c-io-0.14.9/source/windows/iocp/socket.c
--- aws-c-io-0.14.9.orig/source/windows/iocp/socket.c	2024-06-04 18:27:58.000000000 +0200
+++ aws-c-io-0.14.9/source/windows/iocp/socket.c	2024-06-23 10:39:15.047978000 +0200
@@ -9,9 +9,9 @@
 below, clang-format doesn't work (at least on my version) with the c-style comments.*/
 
 // clang-format off
-#include <WS2tcpip.h>
-#include <MSWSock.h>
-#include <Mstcpip.h>
+#include <ws2tcpip.h>
+#include <mswsock.h>
+#include <mstcpip.h>
 // clang-format on
 
 #include <aws/io/socket.h>
diff -urEb aws-c-io-0.14.9.orig/source/windows/shared_library.c aws-c-io-0.14.9/source/windows/shared_library.c
--- aws-c-io-0.14.9.orig/source/windows/shared_library.c	2024-06-04 18:27:58.000000000 +0200
+++ aws-c-io-0.14.9/source/windows/shared_library.c	2024-06-23 10:39:15.047978000 +0200
@@ -4,7 +4,7 @@
  */
 
 // clang-format off
-#include <Windows.h>
+#include <windows.h>
 #include <libloaderapi.h>
 // clang-format on
 
diff -urEb aws-c-io-0.14.9.orig/source/windows/windows_pki_utils.c aws-c-io-0.14.9/source/windows/windows_pki_utils.c
--- aws-c-io-0.14.9.orig/source/windows/windows_pki_utils.c	2024-06-04 18:27:58.000000000 +0200
+++ aws-c-io-0.14.9/source/windows/windows_pki_utils.c	2024-06-23 10:39:15.047978000 +0200
@@ -10,7 +10,7 @@
 
 #include <aws/io/logging.h>
 
-#include <Windows.h>
+#include <windows.h>
 #include <stdio.h>
 #include <string.h>
 
diff -urEb aws-c-io-0.14.9.orig/source/windows/winsock_init.c aws-c-io-0.14.9/source/windows/winsock_init.c
--- aws-c-io-0.14.9.orig/source/windows/winsock_init.c	2024-06-04 18:27:58.000000000 +0200
+++ aws-c-io-0.14.9/source/windows/winsock_init.c	2024-06-23 10:39:15.047978000 +0200
@@ -8,9 +8,9 @@
 below, clang-format doesn't work (at least on my version) with the c-style comments. */
 
 // clang-format off
-#include <WinSock2.h>
-#include <WS2tcpip.h>
-#include <MSWSock.h>
+#include <winsock2.h>
+#include <ws2tcpip.h>
+#include <mswsock.h>
 // clang-format on
 
 #include <aws/io/logging.h>
